src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -375,22 +430,33 @@ def nonhermetic_digest_classpath():
             self.NAILGUN: nonhermetic_digest_classpath,
           })()
 
+          youtline_args = []
+          if youtline:
+            youtline_args = [
+              f"-Xplugin:{self._wartremover_classpath[0]}",
+              "-P:wartremover:traverser:org.wartremover.warts.PublicInference",
```

Comment:
This flag helps reduce the overhead of using compiler plugins
```suggestion
              "-P:wartremover:traverser:org.wartremover.warts.PublicInference",
              "-Ycache-plugin-class-loader:last-modified",
```